### PR TITLE
Shuttle recall fix

### PR DESCRIPTION
**Pull request by Gandalf2k15**
Shuttle can't be recalled when slings ascend or when narsie is summoned.
Admins can now perma call the shuttle.
The shuttle can now be enabled/disabled by admins. 
Demo: https://dl.dropboxusercontent.com/u/109295987/ShareX/2016/06/2016-06-21_23-48-22.mp4
fixes https://github.com/HippieStationCode/HippieStation13/issues/2556

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Fixed my fuckups. Good to go.

**Comment by Gandalf2k15** [comment]
Also demo: https://dl.dropboxusercontent.com/u/109295987/ShareX/2016/06/2016-06-21_23-48-22.mp4

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -658,13 +663,36 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
+	SSshuttle.emergencyNoRecall = 0
 	SSshuttle.emergency.cancel()
 	feedback_add_details("admin_verb","CCSHUT") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 	log_admin("[key_name(usr)] admin-recalled the emergency shuttle.")
 	message_admins("<span class='adminnotice'>[key_name_admin(usr)] admin-recalled the emergency shuttle.</span>")
 
+
 	return
 
+/client/proc/admin_disable_shuttle()
```
Comment (JohnGinnane) [inline code/modules/admin/verbs/randomverbs.dm:675]: I think you should check the mode of the shuttle before assigning to SHUTTLE_STRANDED. Perhaps check for IDLE first, then assign to STRANDED

**Comment by JohnGinnane** [comment]
Code looks pretty good. Merging as it's mostly a bugfix

**Pull request merged**